tests/docs/running-e2e-test.md

```diff
@@ -197,6 +197,67 @@ If you want to run the tests in a similar environment, you can deploy the test i
 
 > Before you run the commands below, ensure that you have an Azure subscription, have the Azure CLI installed, and are logged into Azure (`az login`)
 
+## Run E2E tests using a Wireguard Tunnel with tailscale
+
+[Tailscale](https://tailscale.com/) is a zero-config VPN that provides NAT traversal out-of-the-box allowing our services and pods to be called directly - using its ClusterIP - without needed to be exposed using a loadbalancer.
+
+This provides a few advantages including the decrease of the total test duration.
+
+if you want to run the tests using tailscale as your network, few things are necessary:
+
+1. [Create a tailscale account](https://login.tailscale.com/), this will be necessary since we're going to use personal keys.
+2. [Download and install](https://tailscale.com/download/) the tailscale client for your OS.
+3. When you're logged in, navigate to the menu `Access Controls` and two things are necessary, edit the ACL definition with:
+   1. Create a new tag that will be used later to assign permissions to keys.
+    ```json
+    {...
+      "tagOwners": {
+        "tag:dapr-tests": ["your_email_comes_here@your_domain.com"],
+      }
+    }
+    ```
+   2. Assign permissions to the created tag. Since we are going to use the [tailscale subnet router](https://tailscale.com/kb/1185/kubernetes/), it is much convenient that the subnet router should auto approve the registered routes, for that, use the following acl.
+   ```json
+     {...
+       "autoApprovers": {
+         "routes": {
+           "0.0.0.0/0": ["tag:dapr-tests"],
```

Comment:
Since all Kubernetes routes are on 10.0.0.0/8 can you restrict it to that? For security reasons (so people can’t change the route to, say, google.com) and also to reduce the risk of mistakes.

Reply:
Yes, it makes sense, thanks